.bazelrc

```diff
@@ -97,3 +97,18 @@ build --cxxopt=-DOVMS_DUMP_TO_FILE=0
 # very strict in sandboxing
 test --test_env PYTHONPATH=/opt/intel/openvino/python:/ovms/bazel-bin/src/python/binding
 test --test_env LD_LIBRARY_PATH=/opt/opencv/lib/:/opt/intel/openvino/runtime/lib/intel64/:/opt/intel/openvino/runtime/3rdparty/tbb/lib/
+# genai hardcodes path during build time so without this so is not visible
+test --test_env OPENVINO_TOKENIZERS_PATH_GENAI=/opt/intel/openvino/runtime/lib/intel64/libopenvino_tokenizers.so
+
+build --remote_cache=http://10.123.246.16:9666
```

Comment:
should probably be user defined?

Reply:
# Load rc file with user-specific options.
try-import %workspace%/.bazelrc.user